xtask/src/ci.rs

```diff
@@ -60,7 +60,7 @@ pub enum CiCmd {
     NightlyAll,
     /// Lint default features with clippy (nightly)
     ClippyDefault,
-    /// Lint ruma-common with clippy on a wasm target (nightly)
+    /// Lint wasm features with clippy (nightly)
```

Comment:
"wasm features" is a bit unclear imo, as its definition is very vague. We could maybe say:

```suggestion
    /// Lint client features with clippy on a wasm target (nightly)
```